ArcGISRuntimeSDKQt_QMLSamples/Routing/NavigateARouteWithRerouting/NavigateARouteWithRerouting.qml

```diff
@@ -239,20 +238,21 @@ Rectangle {
             }
         }
 
-        // Output new voice guidance
-        onNewVoiceGuidanceResultChanged: {
-            speaker.textToSpeech(newVoiceGuidanceResult.text);
-        }
+        //        // Output new voice guidance
```

Comment:
don't need to comment the comment. Should there be a comment explaining why we are disabling this bit?